plugins/filter/agent.py

```diff
@@ -1,9 +1,15 @@
 def to_agent_driver_type(data):
     """ Return the desired Sysdig Agent driver type """
     try:
-        return data['agent']['driver']['type']
+        user_input = data['agent']['driver']['type']
+        if user_input == "ebpf":
+            return "legacy_ebpf"
+        if user_input == "kmodule":
+            return "kmod"
+        else:
+            return user_input
```

Comment:
plz add a comment to report that this was added for backward compatibility and we can get rid of it when we bump the major release.